contracts/NiftyFootballTradingCardEliteBlindPack.sol

```diff
@@ -92,7 +96,7 @@ contract NiftyFootballTradingCardEliteBlindPack is Ownable, Pausable {
             generatedTokenIds[i] = _generateAndAssignCard(_to);
         }
 
-        _takePayment();
+        _takePayment(_numberOfCards);
```

Comment:
`totalPrice(_numberOfCards)` is called twice in the life cycle of the method, can we only call it once and save the GAS?